src/test/java/teammates/test/cases/action/InstructorFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -35,15 +35,10 @@ protected String getActionUri() {
     }
 
     @Override
-    protected void prepareTestData() {
-        dataBundle = loadDataBundle("/InstructorFeedbackSubmissionEditSaveActionTest.json");
```

Comment:
This way, both the typical data bundle and `InstructorFeedbackSubmissionEditSaveActionTest.json` will be loaded.